### ISSUE TITLE
Will there be a documentation for using Mapbox Maps SDK v10 with Java Code?

### ISSUE THREAD

**Issue by mrfaa** [mapbox/mapbox-maps-android#1030]
Will there be a documentation for using Mapbox Maps SDK v10 with Java Code?

**Comment by ZiZasaurus** [comment]
@mrfaa the current [API documentation](https://docs.mapbox.com/android/maps/api/10.2.0/mapbox-maps-android/) works for both Kotlin and Java. We have some Java examples [here](https://github.com/mapbox/mapbox-maps-android/tree/main/app/src/main/java/com/mapbox/maps/testapp/examples/java) and also try to make our code base java friendly, however Kotlin as a language is compatible with Java language.

**Comment by mrfaa** [comment]
@ZiZasaurus  thank you very much for your fast answer. Is it planned to present a java code for all the code examples in the guides (https://docs.mapbox.com/android/maps/guides/) and within the examples (https://docs.mapbox.com/android/maps/examples/) ?

**Comment by ZiZasaurus** [comment]
@mrfaa while we will continue to ensure that the SDK can be used by Java, there are no plans to  create the Java versions of our existing Kotlin code snippets or examples.

**Comment by Gango98** [comment]
It's a shame that there is no Java documentation for v10. How can we use Java SDK without knowing anything? The first time I used Mapbox (6 months ago on v9) I was completly lost. I complained about it with support and after all this time the situation is still the same.

If you don't want to support Java developers, throw away all Java implementations and write on your _awarded documentation_ "Only for Kotlin"
